clientconn.go

```diff
@@ -561,6 +569,9 @@ func (csm *connectivityStateManager) updateState(state connectivity.State) {
 		return
 	}
 	csm.state = state
+	if csm.pubSub != nil {
```

Comment:
Since the `pubSub` field is initialized at creation time, we dont need this `nil` check here.